aks-hci/known-issues/0EEFF35A-63D0-4EF2-81D9-5BF5BE25750E.yml

```diff
@@ -4,11 +4,12 @@
 "Reviewer": "abha"
 "ms.topic": "faq"
 "Schema": "doc"
-"Metatitle": "Troubleshoot storage issues and errors in Azure Kubernetes Service on Azure Stack HCI and Windows Server"
-"Metadescription": "Get help to troubleshoot storage issues and errors in Azure Kubernetes Service on Azure Stack HCI and Windows Server."
+"Metatitle": "Troubleshoot storage issues and errors in AKS hybrid"
+"Metadescription": "Get help to troubleshoot storage issues and errors in AKS hybrid."
 "title": "Fix known issues and errors when managing storage"
 "Summary": |
-  Use this topic to help you troubleshoot and resolve storage-related issues in Azure Kubernetes Service on Azure Stack HCI and Windows Server.
+  Use this topic to help you troubleshoot and resolve storage-related issues in AKS hybrid. 
+  Azure Kubernetes Service hybrid deployment options ("AKS hybrid") offer a fully supported container platform that can run cloud-native applications on the [Kubernetes container orchestration platform](https://kubernetes.io/). The architecture supports running virtualized Windows and Linux workloads.
```

Comment:
@DaleP123, why do we need to add this sentence? These articles are short, focused troubleshooting topics, not technical overviews.

Reply:
I have been adding the AKS hybrid description to the YMLs that add the introductory text to each category of known issues.   That has been standard for conceptual and procedural topics. I have no problem with removing it from troubleshooting introductions. If they are looking for troubleshooting info, they know what AKS hybrid is.

Reply:
AKS hybrid into has now been removed from each category introduction that it had been added to.